### ISSUE TITLE
Remove default maxspeed when redundant?

### ISSUE THREAD

**Issue by atorger** [atorger/nvdb2osm#16]
NVDB specifies maxspeed on all roads, ie also the default 70 on roads that have no maxspeed signs. This maxspeed is redundant, and ideally I would not like to have it in there.

However I haven't come up with a reliable method of where to remove the maxspeed and where to keep it, ie where there's actually a sign stating 70 and where's it's just representing the default. I've also noted that this default maxspeed sometimes is 50, when a residential area is nearby, and I have manually corrected it in obvious places when working with the file. My theory is that residential in NVDB is actually a polygon at some level, and get translated into the roads, that's why one can see some strange transitions from residential to unclassified in some situations, and this maxspeed 50 which also may be polygon-based at some level, not totally overlapping with residential though.

Any good ideas for figuring out where maxspeed is an actual sign and where it is default and could be removed would be great.

The current design choice has been "better keep lots of redundant maxspeed than accidentally removing some actual signed maxspeeds", but if you change that to "better avoid redundant maxspeed and pay the price that some actual signed maxspeeds may be removed" we could probably just look at KLASS and make assumptions. I don't think there is a fool-proof method though that would do it 100% correct, but I haven't given it much thought yet

**Comment by NKAmapper** [comment]
I have not seen any data in the DKHastighetsgrans shape file which could indicate if a speed limit is signed or not. I guess you are using the DKTattbebyggtOmrade shape file for the urban/rural information. It is supposed to show where traffic restrictions related to urban areas are in force.

We have had the same discussion in Norway. We have 50 as default maxspeed in urban areas and 80 elsewhere, if there is no sign. However, there is no indication in the Norwegian NVDB of whether maxspeed is signed or not, so we decided to keep both 50 and 80 due to the lack of data, and to help routing engines, which may not be fully updated about national default speeds and which certainly not have data about urban/rural areas.

We decided to not include maxspeed-tagging for private roads because the quality seems quite arbitrary. That removed 70% of all maxspeeds. Quality was top notch for state and county roads. Quality for municipality roads varied, but we decided to keep it because maxspeed is an important factor for routing engines. It will have to be edited by local mappers over the next years.

So one possible proposal would be to remove maxspeed from private roads and forestry roads, but perhaps keep for the "Driftbidrag statligt/Vägnr" roads, which seem to be better maintained.
